tasks/inventory_report/schemas/output.json

```diff
@@ -3,130 +3,114 @@
+            "type": "string"
           },
-          "granules": {
+          "ingestDate": {
+            "description": "Date and time in UTC that the data was originally ingested into ORCA.",
+            "type": "string",
+            "pattern": "^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(.[0-9]*)?Z"
+          },
+          "lastUpdate": {
+            "description": "Date and time in UTC that information was updated.",
+            "type": "string",
+            "pattern": "^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(.[0-9]*)?Z"
+          },
+          "files": {
+            "description": "The files within the granule. Defaults to the latest stored version of each file.",
             "type": "array",
             "items": {
               "type": "object",
               "required": [
-                "id",
-                "cumulusCreateTime",
-                "executionId",
-                "ingestDate",
-                "lastUpdate",
-                "files"
+                "name",
+                "cumulusArchiveLocation",
+                "orcaArchiveLocation",
+                "keyPath",
+                "sizeBytes",
+                "hash",
+                "hashType",
+                "version"
               ],
               "properties": {
-                "id": {
-                  "description": "The unique ID of the granule. Provided by Cumulus.",
+                "name": {
+                  "description": "The name and extension of the file.",
                   "type": "string"
                 },
-                "cumulusCreateTime": {
-                  "description": "Date and time in UTC the granule was ingested.",
-                  "type": "string",
-                  "pattern": "^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(.[0-9]+)?(Z)?\\+00:00$"
+                "cumulusArchiveLocation": {
+                  "description": "Cumulus bucket the file resides in.",
+                  "type": "string"
                 },
-                "executionId": {
-                  "description": "Step function execution ID from AWS.",
+                "orcaArchiveLocation": {
+                  "description": "ORCA S3 Glacier bucket the file resides in.",
                   "type": "string"
                 },
-                "ingestDate": {
-                  "description": "Date and time in UTC that the data was originally ingested into ORCA.",
-                  "type": "string",
-                  "pattern": "^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(.[0-9]+)?(Z)?\\+00:00$"
+                "keyPath": {
+                  "description": "Full S3 path to the file including the file name and extension."

```

Comment:
May want to mention that it is the key only and excludes the `s3://` and bucket name possibly.

Reply:
Ah. Looks like the docs are out of date with the sql file. This'll take a bit.